CherkasovaDA/lab_3/main.cpp

```diff
@@ -0,0 +1,260 @@
+#include <iostream>
+
+using namespace std;
+
+struct MyType
+{
+    int a;
+};
+
+class MyVector
+{
+    int size = 0;
+    int capasity = 0;
+    MyType* massiv = 0;
+   
+    MyType NullItem()
+    {
+        MyType ItemNull{ 0 };
```

Comment:
Not all types have constructor with int (0) as parameter
If you'd like to write more generic code - you can write
`MyType ItemNull{}; // this will call the default constructor`